spec/support/shared/with_progress.rb

```diff
@@ -19,6 +19,24 @@ def start_module(mod)
     view_pages_before(mod, 'module_intro')
   end
 
+  # Visit the first page in first submodule
+  #
+  def start_submodule(mod)
```

Comment:
This suggestion is harking back to what I nearly commented before but I think it's worth doing. In our test suite we likely don't need to have a third module get started yet. If we did, I'd say having `start_submodule` accept a second integer param for the nth intro is wise. The alternative is to rename it to `start_first_submodule` (and also `start_first_topic`), the specs are less ambiguous, the comment is redundant. What do you think to that?

Reply:
Yeah, i think the second option is best. I'll make the changes